db/migrate/20220718210823_create_reviewed_not_spammer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CreateReviewedNotSpammer < ActiveRecord::Migration[5.2]
+  def up
+    create_table :reviewed_not_spammers do |t|
+      t.references :account, foreign_key: true, unique: true
+      t.timestamps
+    end
+
+    execute <<-SQL.squish
+      CREATE VIEW potential_spammers AS
```

Comment:
Will this work without defining it as **oh.potential_spammers** ?

Reply:
Not exactly sure what you mean here, It is coming in oh. potential_spammers as a view. Is there another way I should be doing it?